OgcApi.Net/OgcFeaturesApiServiceCollectionExtensions.cs

```diff
@@ -21,6 +22,14 @@ public static IServiceCollection AddOgcApi(this IServiceCollection services, str
                     });
 
                 if (ogcApiOptions == null) return;
+                foreach (var item in ogcApiOptions.Collections.Items)
+                {
+                    if (item.Tiles is CollectionTilesOptions tiles)
```

Comment:
item.Tiles is CollectionTileOptions by definition. What does this statement is checking for? Is this null check?